src/signed-api-fetch/data-fetcher.ts

```diff
@@ -38,23 +38,21 @@ export const stopDataFetcher = () => {
 /**
  * Calls a remote signed data URL.
  * @param url
+ * @param signedDataFetchInterval
  */
-export const callSignedDataApi = async (url: string): Promise<SignedData[] | null> => {
-  const goAxiosCall = await go<Promise<AxiosResponse>, AxiosError>(
-    async () =>
-      axios({
-        method: 'get',
-        timeout: HTTP_SIGNED_DATA_API_ATTEMPT_TIMEOUT - HTTP_SIGNED_DATA_API_HEADROOM / 2,
-        url,
-        headers: {
-          Accept: 'application/json',
-        },
-      }),
-    {
-      attemptTimeoutMs: HTTP_SIGNED_DATA_API_ATTEMPT_TIMEOUT,
-      totalTimeoutMs: HTTP_SIGNED_DATA_API_ATTEMPT_TIMEOUT + HTTP_SIGNED_DATA_API_HEADROOM / 2,
-      retries: 0,
-    }
```

Comment:
I don't think we need these anymore, right? The `axios` request timeout should be enough to resolve the `go` promise. Let me know if I'm missing something 😅